### DISCUSSION TITLE
Iotas on the ledger are not displayed.

### DISCUSSION THREAD

**Discussion by Theke31** [iotaledger/firefly#8633] [Help & Support]
I've had my Iotas on a Ledger Nano S for many years. Probably missed the migration. Is there a way that I can get the iotas back. I would be very happy if someone could help me. I then pay 500 iotas if successful.

**Comment by begonaalvarezd** [comment]
Please follow to migrate your funds https://blog.iota.org/iota-legacy-migration-tool/
